tests/sync-actor/test_speculative_appeals_actor.py

```
@@ -0,0 +1,172 @@
+
+# The actor bootstraps Django inside a fresh Ray worker (settings, urlconf, app
+# registry) before it can serve anything, and a cold start in CI can take 20+s.
+ACTOR_BOOT_TIMEOUT = 120
+# Generous, but finite on purpose: a bridge that deadlocks inside the actor's
+# event loop fails as a timeout here instead of hanging the suite.
+PREFETCH_TIMEOUT = 120
+
+
+@pytest.mark.django_db
+class TestSpeculativeAppealsActorRay(TransactionTestCase):
+    fixtures = ["fighthealthinsurance/fixtures/initial.yaml"]
```

Ruff (0.15.21)

[warning] Mutable default value for class attribute

(RUF012)